### DISCUSSION TITLE
[Achievement Request]: Issue Muncher

### DISCUSSION THREAD

**Discussion by mdanish-kh** [community/community#35579] [Other Feature Feedback, Questions, & Ideas]
Can't think of a better name, but I'll leave the creative naming to the community.

### Proposal
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue
As GitHub has support for closing an issue through a linked PR, a useful achievement would be to keep track of how many issues one has helped to close through linked PRs. The credit for closing the issue for this achievement should be given to the author of the PR not the one who merges it. (I say this because the current behavior is that the issue is closed with the message: `<Merger_name> closed this issue in <PR#>`). I believe this would be a better achievement that adds value to a person's profile and acknowledges one's contributions more appropriately than other achievements.

**Comment by ArnabDey1543** [comment]
> Can't think of a better name, but I'll leave the creative naming to the community.
> 
> ### Proposal
> https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue As GitHub has support for closing an issue through a linked PR, a useful achievement would be to keep track of how many issues one has helped to close through linked PRs. I believe this would be a better achievement that adds value to a person's profile and acknowledges one's contributions more appropriately than other achievements.

Link Tracker
